### ISSUE TITLE
Longitude & latitude attributes of stream entity break map in sidebar - C210

### ISSUE THREAD

**Issue by Xlinx64** [JurajNyiri/HomeAssistant-Tapo-Control#302]
### Description

Because the HD and SD stream have Longitute and latitude values in the attributes they are messing with the map in the sidebar. The map is completely zoomed out and you see the whole world. 
As a test I removed the values from the attribute via the developer settings and the map started to work again.
It would already help to rename these values.

### Reproduction Steps

1. Enable the HD or SD Stream in Home assistant.
2. Via the left pane open the Map
3. The map is completely zoomed out and unusable

### Expected behavior

The stream entity should not mess with the map.

### Device Firmware

1.3.0 Build 220830 

### Using stream component

Yes

### Does camera work via official integrations?

N/A

### Camera has all attributes filled out in developer tools

Yes

### HASS Environment

Home Assistant OS on ODROID N2+

### Search for similar issues

Yes

### Additional information

![Bildschirm­foto 2023-03-28 um 15 37 24](https://user-images.githubusercontent.com/14150988/228255462-92b29777-e980-4ccb-bd52-398ea2f2a7a9.png)

**Comment by JurajNyiri** [comment]
What camera is this? My C200s do not expose anything like this. Is this possible to be set up via the Tapo app?

**Comment by Xlinx64** [comment]
The camera is set up via the Tapo app.
The properties look like this

> longitude: 164903
> latitude: 484042

**Comment by Taomyn** [comment]
I confirm the same behaviour, I have both a C200 and C210 and if the C210 is enabled the map behaves as described.

**Comment by AndreiArdelean1** [comment]
I have the same issue with C225

**Comment by skgsergio** [comment]
Managed to get rid of the location, I've rejected Tapo App location permissions, deleted the camera from Tapo app, and re-added it. Then I reconfigured Tapo Control in Home Assistant.

Looks like it sets the location when you configure the camera in Tapo app, which makes sense to me since this camera is near a window and the other is deep inside the house where there is no GPS signal and was the first I've configured.
